doc/lua_api.md

```diff
@@ -5431,6 +5431,8 @@ Utilities
       lsystem_decoration_type = true,
       -- Overrideable pointing range using the itemstack meta key `"range"` (5.9.0)
       item_meta_range = true,
+      -- Allow passing in an optional "actor" ObjectRef into node interaction calls (5.9.0)
```

Comment:
This should mention the affected functions.

Reply:
Resolved in 391f46c62105d536b0fef43a62767c10b95a21a1